### ISSUE TITLE
Truth table import problem

### ISSUE THREAD

**Issue by tkoenig1** [reds-heig/logisim-evolution#517]
When importing the following truth table
```
a0 b0 sel2 sel1 sel0 | out0
~~~~~~~~~~~~~~~~~~~~~~~~~~~
0 0 1 - 0 | 1
1 1 - 1 - | 1
1 0 - - 1 | 1
0 1 - - 1 | 1
0 - 0 0 - | 1
0 0 1 1 1 | -
0 1 1 1 1 | -
1 0 1 1 1 | -
1 1 1 1 1 | -
0 0 0 1 - | 0
0 0 1 0 1 | 0
1 0 - - 0 | 0
0 1 - 1 0 | 0
0 1 1 - 0 | 0
1 1 - 0 - | 0
```
the program complains "Some inputs are repeated. For example, rows 4 and 5 have overlapping input values 0 1 - - 1 and 0 - 0 0 -." Ignore erros and try again?

While this is true, the truth table is nevertheless valid, because both rows lead to the same result, 1.

It would be nice to do that additonal test in that case, too.

**Comment by maehne** [comment]
Thanks for reporting this issue! Pull requests to fix this issue are welcome! For the coming months we'll have less bandwidth on our side to push forward the development due to a busy spring term.

**Comment by hewzhew** [comment]
I retested this on the current code using the table from the original report. The text-table import now succeeds without reporting overlapping inputs, and the analyzer model is populated with 5 input bits, 1 output bit, and 15 visible rows.

It looks like the current `TruthTable.setVisibleRows()` logic already allows overlapping input rows when their defined output values are compatible. Could this issue be closed as already fixed?

**Comment by maehne** [comment]
Indeed, this issue was fixed by PR #1717 and we forgot to close this issue. @tkoenig1: Please reopen if you should find still cases where this issue is not resolved.
